CHANGELOG.md

```diff
@@ -5,7 +5,32 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 ### Added
+- Bump sensu-plugin runtime dep to 3.0 with Sensu Go to Sensu Core 1.x migration option (@jspaleta)
 - Ruby 2.4.1 testing
+- Ruby 2.5.1 testing (@jspaleta)
+- Added --verbose cmdline switch to aid in local testing
+- Added --disable_send cmdline switch to aid in local testing
+
+### Breaking Changes
+- Dropped 2.0 ruby testing (@jspaleta)
+- Dropped 2.1 ruby testing (@jspaleta)
+- Dropped 2.2 ruby testing (@jspaleta)
+- Set required_ruby_version in gemspec to >= 2.3.0 (@jspaleta)
+
+### Changed
+- Bump twilio runtime dep to 5.17.0 (@jspaleta)
+- Bump rest-client runtime dep to 2.0.2 (@jspaleta)
```

Comment:
what uses `rest-client` from what I could tell that may be an unused dependency. Though #1 would suggest that not be the case. I will try to take a closer look at the code tomorrow to make sure this is not a breaking change.

Reply:
hmm not sure.. i'll try to test with the dep removed using my twilio creds.

Reply:
rest-client is an needed dep by twilio-ruby. Trying to post to twilio with rest-client not installed results in error

Reply:
nope unrelated error.  removing rest-client now.  Twilio-ruby internals changed. Fixed with next commit